misc/blobstore-signed-urls.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/blobstore/agent/user
+  value: not_used_but_required_by_agent_using_singed_urls
```

Comment:
Can we change the dummy values to something like `not_used_but_required_by_some_cpis`? The values aren't required by the agent actually. This is misleading a little bit.

Reply:
Indeed, now that I've dug more into the subject, this appears as being definitely relevant. So, yes! We can definitely do that.